Cargo.toml

```diff
@@ -11,7 +11,7 @@ license = "Apache-2.0"
 [dependencies]
 async-trait = "0.1"
 axum = { version = "0.7" }
-chrono = { version = "0.4", default-features = false }
+chrono = { version = "0.4.38", default-features = false }
```

Comment:
It's better be less strict with deps versions in libraries, e.g. I don't see a point forcing every user to have `chrono >=0.4.38` if the library still works with `0.4.0`